bc_obps/reporting/api/report_additional_data.py

```diff
@@ -52,8 +50,9 @@ def save_report_additional_data(
 @handle_http_errors()
 def get_report_additional_data_by_version_id(
     request: HttpRequest, report_version_id: int
-) -> Tuple[Literal[200], ReportAdditionalDataOut]:
-    report_person_responsible = ReportAdditionalDataService.get_report_person_responsible_by_version_id(
+) -> tuple[Literal[200], ReportAdditionalData]:
+    report_additional_data = ReportAdditionalDataService.get_report_report_additional_data_by_version_id(
```

Comment:
I'd say the name is redundant here, the "by version id" is already exposed by the name of the parameter.
I'd suggest `ReportAdditionalData.get_report_additional_data` to match the `save` function name